locales/en-US/app.ftl

```diff
@@ -926,29 +926,48 @@ TransformNavigator--collapse-indirect-recursion = Collapse indirect recursion: {
 #   $item (String) - Name of the function that transform applied to.
 TransformNavigator--collapse-function-subtree = Collapse subtree: { $item }
 
-## Source code view in a box at the bottom of the UI.
+## "Bottom box" - a view which contains the source view and the assembly view,
+## at the bottom of the profiler UI
 
-# Displayed while the source view is waiting for the network request which
-# delivers the source code.
+# Displayed while a view in the bottom box is waiting for code to load from
+# the network.
 # Variables:
 #   $host (String) - The "host" part of the URL, e.g. hg.mozilla.org
-SourceView--loading-url = Waiting for { $host }…
+BottomBox--loading-url = Waiting for { $host }…
```

Comment:
Is this change strictly necessary? We're asking to retranslate 3 strings, which seems unnecessary, since "SourceView" seems still applicable.

Also, one of the SourceView string remains around.

Reply:
It's not strictly necessary. I can change it to keep the old name.

Reply:
I've changed the names back.